packages/harmony/src/components/input/TextInput/HelperText.module.css

```diff
@@ -0,0 +1,3 @@
+.root {
+  margin-top: var(--harmony-spacing-xs);
```

Comment:
This feels weird? Doesn't the root input have a flex gap to handle this?

Reply:
There wasn't a gap before but I definitely prefer that so I moved it over and yoinked this whole component